tests/v2/integration/catalogv2/rancher_managed_charts_test.go

```diff
@@ -127,6 +127,7 @@ func (w *RancherManagedChartsTest) resetSettings() {
 }
 
 func TestRancherManagedChartsSuite(t *testing.T) {
+	t.Skip()
```

Comment:
I don't think we need to skip the whole suite here
https://github.com/igomez06/rancher/commit/fb27c8240f86e37cd24cfde142641030153c35df#diff-fd1dcaa06d6248e12167940fa5128712872e50b2709831055cb2414c445157ecR64

I'm pretty sure the lower in the poll interval should fix it.

Reply:
and only skip the `TestRancherManagedChartsSuite/TestServeIcons` test